### ISSUE TITLE
Support for commenting out properties when in 1 class per multine line

### ISSUE THREAD

**Issue by jonsherrard** [schoero/eslint-plugin-better-tailwindcss#199]
When using `cmd  + /` to comment out a line in with this config.

```
 rules: {
    'better-tailwindcss/multiline': [
      'error',
      {
        classesPerLine: 1,
        group: 'newLine',
        printWidth: 0,
        // preferSingleLine: true,
      },
    ],
  },
```

This component:

```tsx
<View
  className={`
    mb-4
      flex-row
      items-center
      lg:mb-6
  `}
>
```

will be sorted to:

```tsx
<View
  className={`
    mb-4
    flex-row
    //
    items-center
    lg:mb-6
  `}
>
```

```diff
<View
  className={`
     mb-4
     flex-row
-    items-center
+    //
+    items-center
     lg:mb-6
  `}
>
```

I actually wrote my own prettier plugin for multiline sorting in tailwind until i found this much better plugin. 

Issues I had with implementation:
- "look ahead" of comments  to get the property
- having the `// ` "stick" the property in sorting
- Performance (I was regexing every property which was probably clumsy as hell

**Comment by schoero** [comment]
Correct me if I'm wrong, but comments do not work in string literals right? I could make the comment stick to the current line, but what you are proposing doesn't have the desired effect.
